presidio-anonymizer/app.py

```diff
@@ -57,8 +57,10 @@ def anonymize():
             if not content:
                 return ErrorResponse("Invalid request json").to_json(), 400
 
-            data = AnonymizerRequest(content, self.engine.builtin_anonymizers)
-            text = self.engine.anonymize(data)
+            anonymizers = AnonymizerRequest.handle_anonymizers_json(content)
+            analyzer_results = AnonymizerRequest.handle_analyzer_results_json(content)
+            text = self.engine.anonymize(content.get("text"), analyzer_results,
```

Comment:
nit: Consider passing arguments by name and not position. More robust towards future changes. e.g.
```python
text = self.engine.anonymize(text=content.get("text"), 
                             analyzer_results=analyzer_results, 
                             anonymizers_config=anonymizers)
```

Reply:
Done.